syntax = "proto3";
package dfi;

message DFICreateFlowResponse {
	uint32 return = 1;
}
